modules/statics/src/base.ts

```diff
@@ -719,6 +719,13 @@ export enum UnderlyingAsset {
   CHEX = 'chex',
   IQ = 'iq',
   EOS_BOX = 'eos:box',
+
+  // Avax Token ERC-20
+  QI = 'qi',
+  XAVA = 'xava',
+  KLO = 'klo',
+  JOE = 'joe',
+  PNG = 'png',
```

Comment:
after some discussion with @CapnMigraine, we should add them as follows:

```suggestion
avaxc:QI = 'avaxc:qi',
```

Reply:
Question, the other tokens were removed or should be modified the same way than "qi" ?

Reply:
If the asset exists on two chains, then the tokens which represent it on each chain should share the same underlying asset

Reply:
Done